react/Checkbox/Checkbox.js

```diff
@@ -22,10 +22,10 @@ export default class Checkbox extends Component {
 
   static propTypes = {
     id: PropTypes.string.isRequired,
-    label: PropTypes.string.isRequired,
```

Comment:
I don't feel it's a right thing to do as one day someone will come and delete it, thinking `what a stupid thing they've added here`